eta/core/video.py

```diff
@@ -1908,7 +1888,22 @@ def sample_select_frames(
         tmp_patt = os.path.join(d, "frame-%d" + ext)
         ffmpeg = FFmpeg(
             size=size, out_opts=["-vf", select_arg_str, "-vsync", "0"])
-        ffmpeg.run(video_path, tmp_patt)
+
+        #
+        # Make sure to handle case where ffmpeg command errors out.
+        # This can happen if the `frames` argument passed to the current
+        # function includes frames that are not actually present in the video.
+        #
+        try:
+            ffmpeg.run(video_path, tmp_patt)
+        except etau.ExecutableRuntimeError as e:
```

Comment:
I'm not comfortable approving this as a fix until we understand what exactly was causing problems in the first place. How do we know that switching to `VideoProcessor` will change anything? If the problem is that `ffmpeg` can't parse some frame of the video, that will be true with both approaches.

Secondly, if failures in this ffmpeg command will result in a partially populated temp dir, then that's all we want anyway. There is no need to redo all the expensive computation with `VideoProcessor` just to achieve the same result.

As you can tell, I don't like approving bandaid PRs in our core methods in ETA. We need bulletproof code here!